selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/SeleniumWebDriverHelper.java

```diff
@@ -469,7 +473,7 @@ public boolean isElementVisible(By locator) {
    * @param webElement
    * @return state of element visibility
    */
-  public boolean isElementVisible(WebElement webElement) {
+  public boolean isVisible(WebElement webElement) {
```

Comment:
Looks like the method is waiting on element visibility with timeout ATTACHING_ELEM_TO_DOM_SEC.
Let's reflect it in method name and java docs.

Reply:
How about `isVisibileDuringTimeout` name?